sceptre/scipool/config/develop/sc-product-ec2-linux-docker-notebook.yaml

```diff
@@ -0,0 +1,18 @@
+template:
+  path: "sc-product-ec2-linux-docker-notebook.j2"
+stack_name: "sc-product-ec2-linux-docker-notebook"
+stack_tags:
+  OwnerEmail: "it@sagebase.org"
+dependencies:
+  - "develop/sc-portfolio-ec2.yaml"
+  - "develop/sc-ec2-actions.yaml"
+  - "develop/sc-product-ec2-linux-notebook-write-to-ssm-policy.yaml"
+parameters:
+  ReplaceProvisioningArtifacts: "true"
+sceptre_user_data:
+  # force cloudformation to update stack by setting a random number to the latest product's description
+  ProvisioningArtifactParameters: |
+    - Description: 'Update instance types {{ range(1, 10000) | random }}'
+      Info:
+        LoadTemplateFromURL: 'https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/{{stack_group_config.service_catalog_library}}/v1.0.0/ec2/sc-ec2-linux-docker-notebook.yaml'
+      Name: 'v0.0.1'
```

Comment:
as a convention we match this version with the one that in `LoadTemplateFromURL`, so should be `v1.0.0`

Reply:
Done.